src/components/images_uploading/UploadImageFile.js

```diff
@@ -1,54 +1,93 @@
+//Unable to make it happen with Redux - might try to refactor to Redux way later
+
 import React, { Component } from "react";
 import axios from "axios";
-import DisplayImage from "./../images_uploading/DisplayImage";
+
 class UploadImageFile extends Component {
   state = {
-    selectedFile: null
-  };
-
-  fileSelectedHandler = event => {
-    this.setState({
-      selectedFile: event.target.files
-    });
+    selectedFile: null,
+    lot: 0,
+    unitNumber: "",
+    productDescription: ""
   };
 
   fileUploadHandler = event => {
     event.preventDefault();
-    const data = new FormData();
+    const { lot, selectedFile, unitNumber, productDescription } = this.state;
+    const formData = new FormData();
 
-    for (let x = 0; x < this.state.selectedFile.length; x++) {
-      data.append("file", this.state.selectedFile[x]);
+    //At this moment, if create apt without attaching an img, the app will break. Need to fix it later.
+    for (let x = 0; x < selectedFile.length; x++) {
+      formData.append("file", selectedFile[x]);
+      formData.append("lot", lot);
+      formData.append("unitNumber", unitNumber);
+      formData.append("productDescription", productDescription);
     }
 
-    console.log(this.state.selectedFile);
+    console.log(this.state); //Keep this line to see if upload successful in browser console
 
     axios
-      .post(`http://localhost:5000/images/upload`, data, {
-        onUploadProgress: progressEvent => {
-          console.log(
-            "Upload Progress: " +
-              Math.round((progressEvent.loaded / progressEvent.total) * 100) +
-              "%"
-          );
-        }
-      })
+      .post(`${process.env.REACT_APP_BASEURL}/images`, formData)

```

Comment:
This could be rewritten using LocalApi: 
LocalApi.post('/images', formData)
...